### DISCUSSION TITLE
Is it possible to find AI models information through the API?

### DISCUSSION THREAD

**Discussion by qiakai** [community/community#137480] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm planning to use GitHub Model in the command line, and I'm hoping to find a way to get data from the [GitHub Marketplace](https://github.com/marketplace/models).

**Comment by sedna16** [comment]
would like to know this as well

**Comment by qiakai** [comment]
already  exists:
https://docs.github.com/en/rest/models/catalog?apiVersion=2022-11-28#list-all-models
